### ISSUE TITLE
Текущие показатели: 

### ISSUE THREAD

**Issue by khabibulinmi** [bibatn/measure_segmentation#1]
accuracy =  0.9613436354058129
iou, total_iou, mean_iou
0.53 0.83 0.00 0.56 0.75 0.86 0.00 0.56 0.46 0.45 0.69] 0.84 0.52

**Comment by khabibulinmi** [comment]
nami_classes_to_cvat_labels = {
            'unlabeled': ['background'],
            'road': ['Дорога'],
            'sidewalk': ['Обочина'],
            'person': ['Человек', 'Пешеход', 'person'],
            'vehicle': ['ТС', 'Машины', 'Машина', 'средство', 'Транспортное',
                        'car', 'Поезда', 'Автобусы', 'Грузовики'],
            'bicycle': ['Велосипедист', 'Велосипед'],
            'moto': ['Мотоциклы', 'Мотоциклист'],
            'light': ['Сфетофор', 'Светофор'],
            'sign': ['Знак'],
            'zebra': ['Зебра'],
            'dashed': ['Прерывистая линия'],
            'solid': ['Сплошная линия'],
            'doublesolid': ['Двойная сплошная линия'],
        }

'человек',
'транспортное средство',
'велосипед',
'свет',
'знак',
'дорога',
'мото',
'зебра',
'пунктирная',
'сплошная',
'двойная сплошная'

**Comment by bibatn** [comment]
/mnt/data/VideoAi/Khabibulin/DATA

**Comment by khabibulinmi** [comment]
accuracy =  0.7704184459427655
0.33 0.63 0.00 0.27 0.53 0.23 0.00 0.37 0.07 0.22 0.13] 0.29 0.25

**Comment by khabibulinmi** [comment]
accuracy =  0.8893431507090412
0.60 0.86 0.00 0.58 0.75 0.79 0.00 0.52 0.47 0.43 0.50] 0.78 0.50

**Comment by khabibulinmi** [comment]
СF test
accuracy =  0.8951294654238541
0.52 0.85 0.00 0.41 0.78 0.79 0.00 0.51 0.65 0.66 0.55] 0.79 0.52

**Comment by khabibulinmi** [comment]
test_dataset - 6829; valid_dataset - 2001
CF_short test
accuracy =  0.7902565611169693
0.39 0.65 0.00 0.26 0.53 0.27 0.00 0.18 0.14 0.34 0.23] 0.33 0.27

**Comment by MaratKhabibulin** [comment]
^ 300 эпох обучения, на какой остановка была неизвестно.

**Comment by MaratKhabibulin** [comment]
accuracy =  0.9558259477006629
0.48 0.80 0.00 0.66 0.70 0.86 0.00 0.62 0.39 0.52 0.69] 0.82 0.52

**Comment by MaratKhabibulin** [comment]
^ (3 повторяющиеся картинки с CF_SHORT)
print (len(train_dataset), len(valid_dataset))
7395 2191

**Comment by khabibulinmi** [comment]
70 эпох

**Comment by khabibulinmi** [comment]
accuracy =  0.9604781739255215
0.53 0.86 0.00 0.68 0.72 0.87 0.00 0.67 0.39 0.55 0.73] 0.85 0.54

**Comment by khabibulinmi** [comment]
63+70 эпох
